components/server/src/server.rs

```diff
@@ -147,12 +150,14 @@ pub unsafe fn run_tikv(config: TiKvConfig) {
                 )),
             };
 
-            let proxy_helper = RaftStoreProxyFFIHelper::new(&proxy);
+            let mut proxy_helper = RaftStoreProxyFFIHelper::new(&proxy);
```

Comment:
I think `proxy_helper` should be immutable. You can try like
```suggestion
            let proxy_helper = {
                let mut proxy_helper = RaftStoreProxyFFIHelper::new(&proxy);
                proxy_helper.fn_server_info = Some(ffi_server_info);
                proxy_helper
            };
```
or make another function to implement.